### PR TITLE
consolidate kubectl/oc/virtctl RunCommand() into one

### PR DESCRIPTION
**Pull request by shiywang**
Signed-off-by: Shiyang Wang <shiywang@redhat.com>

**What this PR does / why we need it**:
make downstream kubevirt-ansible more easy to consume the lib.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by shiywang** [comment]
@fabiand @cynepco3hahue Okay with this change ?

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
I can see test `[Fail] HookSidecars VMI definition with SM BIOS hook sidecar [It] should update domain XML with SM BIOS properties` started to fail with
```
level=error timestamp=2018-08-06T12:04:24.185895Z pos=utils.go:1444 component=tests output="Unable to use a TTY - input is not a terminal or the right kind of file\nerror: failed to get domain 'Unable to use a TTY - input is not a terminal or the right kind of file'\nerror: Domain not found: no domain with matching name 'Unable to use a TTY - input is not a terminal or the right kind of file'\n\ncommand terminated with exit code 1\n" reason="exit status 1" msg="kubectl command failed: /var/lib/stdci/container-slaves/1/home/jenkins/workspace/kubevirt_kubevirt_standard-check-pr/kubevirt/cluster/k8s-1.11.0/.kubectl exec -ti --namespace kubevirt-test-default virt-launcher-testvmixg6hz-r7rfh --container compute -- virsh dumpxml Unable to use a TTY - input is not a terminal or the right kind of file,"
• [SLOW TEST:77.333 seconds]
```
@shiywang Can you please check it?

**Comment by shiywang** [comment]
@cynepco3hahue I fixed `[Fail] HookSidecars VMI definition with SM BIOS hook sidecar [It] should update domain XML with SM BIOS properties` this problem, could you please take a look ?

File: tests/utils.go [hunk 1]
```diff
@@ -1441,14 +1436,19 @@ func RunOcCommand(args ...string) (string, error) {
 		args = append(args, "--server", master.String())
 	}
 
-	cmd := exec.Command(KubeVirtOcPath, args...)
+	cmd := exec.Command(cmdPath, args...)
 	kubeconfEnv := fmt.Sprintf("KUBECONFIG=%s", kubeconfig.String())
 	cmd.Env = append(os.Environ(), kubeconfEnv)
 
-	stdOutErrBytes, err := cmd.CombinedOutput()
+	switch cmdName {
```
Comment (shiywang) [inline tests/utils.go]: only change is here, seems `oc` and `kubectl` have to use different cmd Ouput to deal with errors.

**Comment by shiywang** [comment]
ci test please

**Comment by shiywang** [comment]
I saw couple prs have this problem, @cynepco3hahue do you know how to make the CI pass ? test flake ?
```
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] 
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] Ran 152 of 0 Specs in 126.444 seconds
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] FAIL! -- 0 Passed | 152 Failed | 0 Pending | 0 Skipped
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] --- FAIL: TestTests (126.44s)
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] FAIL
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] make: *** [functest] Error 1
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] + release_windows_lock
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] + [[ -e /tmp/download_windows_image.lock ]]
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] + make cluster-down
15:29:59 [check-patch.k8s-1.11.0-release.el7.x86_64] ./cluster/down.sh
```

**Comment by cynepco3hahue** [comment]
Looks like all failures does not really relate to the PR.

**Pull request merged**